DependencyInjection/MonologExtension.php

```diff
@@ -203,6 +203,8 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                     $handler['bubble'],
                     $handler['verbosity_levels'] ?? [],
                     $handler['console_formatter_options'],
+                    null,
+                    $handler['interactive_only'],
```

Comment:
This will be false by default and they will get added even if your Symfony Monolog bridge is too old, but it doesn't hurt, they'll just get ignored.